### PR TITLE
app-emulation/lxd: bump to v2.19 

### PR DESCRIPTION
**Pull request by hsoft**
Two notable changes in the ebuild:

1. explicit branch comment for gopkg.in vendor dependencies because
inconshreveable doesn't follow the branching convention and I couldn't
build without changing the way gentoo-ego-vendor-update works.

2. Remove versions from conf, initd and service filenames. They don't change
often enough to warrant the constant noise of copying them.

### CODE REVIEW

File: app-emulation/lxd/files/lxd.initd [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+DAEMON=/usr/sbin/lxd
```
Comment (monsieurp) [inline app-emulation/lxd/files/lxd.initd]: this line should be defined in as the `command` parameter according to `openrc-run (8)`.

File: app-emulation/lxd/files/lxd.initd [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+PIDFILE=/run/lxd.pid
```
Comment (monsieurp) [inline app-emulation/lxd/files/lxd.initd]: ditto, define it in as the `pidfile` parameter. have a loot at `openrc-run (8)`.

File: app-emulation/lxd/files/lxd.initd (hunk 1 above)
Comment (hsoft) [reply app-emulation/lxd/files/lxd.initd]: @monsieurp I didn't write this file and I'm not familiar with writing init scripts. Do you know of an existing package that has proper init script?

File: app-emulation/lxd/files/lxd.initd (hunk 2 above)
Comment (titanofold) [reply app-emulation/lxd/files/lxd.initd]: DAEMON and PIDFILE are fine in this case. We wouldn't be able to get rid of any of the functions even if we did.

**Comment by asarubbo** [comment]
merged, thanks

**Pull request closed without merging**